Phoenix/Core/Source/Voxels/Blocks.cpp

```diff
@@ -40,13 +38,11 @@ RegisteredBlock::~RegisteredBlock() = default;
 
 BlockRegistry::BlockRegistry() : i(0) {};
 
-BlockRegistry::~BlockRegistry() {};
-
 int BlockRegistry::registerBlock(const std::string& uniqueName,
                                  const std::string& displayName)
 {
 	i++;
-	Blocks[i] = RegisteredBlock(uniqueName, i, displayName);
+	Blocks.push_back(RegisteredBlock(uniqueName, i, displayName));
```

Comment:
Ah, so it looks like you're using `i` as a unique identifier for the blocks - it may make more sense to rename that to `m_uniqueBlockIdCounter` or something along those lines.

Reply:
So much typing (ˊ̥̥̥̥̥ ³ ˋ̥̥̥̥̥)